### ISSUE TITLE
listener displays "not supported"  on W10 Pro 21H1 19043.1083

### ISSUE THREAD

**Issue by formetx** [stascorp/rdpwrap#1483]
Hello Community

i am getting nuts by trying to make RDPWrapper working on my W10 Pro 21h1 10943.1083
The listener seems to works, but it displays the message "not supported" which i dont understand exactly.
the rdpwrap.ini is new and actual.

when i try to connect a client with an rdp session, it connects but says that there is already one user logged onto the RDPWrap server and this one needs to be disconnected.

so basically rdpwrap does not work and i canno have multiple sessions still.....

does anyone have maybe any idea how to fix the issue ?
thank you very much indeed !

TFO

![IMG_5373](https://user-images.githubusercontent.com/84029163/127515977-da87590d-e73b-4240-a3da-c6d033f53515.jpg)

**Comment by pnz1024mb** [comment]
Works on 10 21H1 19043.1110
[rdpwrap_10.0.19043.1110.zip](https://github.com/stascorp/rdpwrap/files/6911781/rdpwrap_10.0.19043.1110.zip)

**Comment by Mannshoch** [comment]
@pnz1024mb @binarymaster 
The last release is from 2017. Any plan to create a new one?

**Comment by formetx** [comment]
I must say: thank you very much !

I have tried immediately after coming back from Holidays
And it works

Respect and best regards
Bravo !


Von: pnz1024mb ***@***.***>
Antworten an: stascorp/rdpwrap ***@***.***>
Datum: Samstag, 31. Juli 2021 um 22:22
An: stascorp/rdpwrap ***@***.***>
Cc: formetx ***@***.***>, Author ***@***.***>
Betreff: Re: [stascorp/rdpwrap] listener displays "not supported" on W10 Pro 21H1 19043.1083 (#1483)


Works on 10 21H1 19043.1110
rdpwrap_10.0.19043.1110.zip<https://github.com/stascorp/rdpwrap/files/6911781/rdpwrap_10.0.19043.1110.zip>

**Comment by Dan245610** [comment]
i am also on win 10 pro and am having the same problem i downloaded the link above but i have no idea what i do with it from here

**Comment by johnwmail** [comment]
> 
> 
> i am also on win 10 pro and am having the same problem i downloaded the link above but i have no idea what i do with it from here

But why you need it?  Is not win 10 pro support built-in support RDP server?

**Comment by sebaxakerhtc** [comment]
same as https://github.com/stascorp/rdpwrap/issues/1447
